UniAtHome/UniAtHome.BLL/Services/AuthService.cs

```diff
@@ -82,7 +82,7 @@ public async Task<LoginResponse> LoginAsync(LoginRequest request)
 
         public async Task<TokenRefreshResponse> RefreshTokenAsync(TokenRefreshRequest request)
         {
-            User user = await usersRepository.FindByEmailAsync(request.Email);
+            User user = await usersRepository.FindByEmailAsync(request.AuthToken);
```

Comment:
I didn't configure copy-pasted and not used class. This change is only for successful build. If you decide to use this class instead of the working one, you'll need to make changes here as well.